### PR TITLE
modals.less: style issue with flex positioning fixed

### PR DESCRIPTION
**Pull request by vied12**
I removed these lines to fix this issue we have on Liveblog:
modal

### CODE REVIEW

**Comment by plamut** [comment]
LGTM, but let's give the designers the final say. :) @pavlovicnemanja92 @Aca-jov

**Comment by Aca-jov** [comment]
:+1:

File: client/app/styles/less/modals.less [hunk 1]
```diff
@@ -62,10 +62,6 @@ body:not(:-moz-handler-blocked) .modal{
 		max-height: 100%;
 	}
 	.modal-content {
-		display: flex;
-		justify-content: center;
-		align-items: flex-start;
-		flex-direction: column;
 		flex: 1 1 auto;
```
Comment (petrjasek) [inline client/app/styles/less/modals.less]: i guess this line is obsolete too

Comment (vied12) [inline client/app/styles/less/modals.less]: It's needed to fill the parent

Comment (petrjasek) [inline client/app/styles/less/modals.less]: I see (me too)

**Pull request merged**